doc/nrf/dm_ncs_distro.rst

```diff
@@ -0,0 +1,85 @@
+.. _dm_ncs_distro:
+
+Redistributing NCS
+##################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This section is intended for users that would like to re-distribute the |NCS|
+with additions or changes to it.  This can be useful if you are a software or
+hardware vendor that would like to augment the |NCS| with additional
+functionality, while at the same time still benfitting from the infrastructure
```

Comment:
nit. `benfitting` should be `benefiting`

Reply:
Thanks!